### ISSUE TITLE
[OJS] ORCID iD icon display requires ORCID Profile Plugin to be enabled

### ISSUE THREAD

**Issue by davidjb** [pkp/pkp-lib#6959]
**Describe the bug**
Article views in OJS display an author's ORCID iD, but the ORCID icon doesn't display unless the ORCID Profile Plugin is installed and enabled, meaning core OJS is silently depends on this plugin.  Could the icon be moved into the core? 

**To Reproduce**
1. Create an article with an author with an ORCID iD attached
2. View the article page and note no icon
3. Enable the ORCID Profile Plugin and reload the article page; note the icon now appears

**What application are you using?**
 OJS 3.3.0.4

**Additional information**

* Article template reference: https://github.com/pkp/ojs/blob/8b86bbca41c27862618024dde7e248b9b5336354/templates/frontend/objects/article_details.tpl#L122
* ORCID Profile Plugin provides `orcidIcon` variable: https://github.com/pkp/orcidProfile/blob/1bfc0d26a5b339080585910aeb48f4c35f4745a9/OrcidProfilePlugin.inc.php#L344

**Comment by NateWr** [comment]
Is there a way to enter an ORCID for an author if not using the ORCID profile plugin? The default templates are coded to support some very common plugin data when it is available (eg - DOIs, How to Cite).

**Comment by davidjb** [comment]
Yes, user profiles and author details associated with submissions/articles have a text input field for ORCID iD by default without the plugin enabled.

When the ORCID Profile plug-in is enabled, the latter (haven’t checked the former) gets replaced with an authorisation Oauth workflow for ORCID - checkboxes appear for emailing the authors to use ORCID’s API to validate their iDs.

**Comment by NateWr** [comment]
> That said, it might be worth OJS looking at changing or removing the default manual entry fields for ORCID iDs given the guidance: “Do not allow users to copy or type in their ORCID iD”.

Thanks  @davidjb, I think there's been some discussion about that but I couldn't find anything in the [plugin repository's issue list](https://github.com/pkp/orcidProfile/issues).

**Comment by davidjb** [comment]
@NateWr I opened #6971 as a feature request - at the moment, the two different processes (manual entry vs ORCID Oauth process) is creating some confusion for journals we host; having just the official process would be ideal.
